File: Data Structures & Algorithms/duplicate-integer/submission-0.java

class Solution {
    public boolean hasDuplicate(int[] nums) {
         HashMap<Integer,Integer> map = new HashMap<>();
        for (int key : nums) {
            if (map.containsKey(key)) {
                return true;
            }
            map.put(key, 1);
        }

        return false;
    }
}

File: Data Structures & Algorithms/find-pivot-index/submission-0.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
        for(int i=0;i<nums.size();i++)
        {
            int leftSum=0;
            for(int j=0; j<i;j++)
            {
                leftSum+=nums[j];
            }
            int rightSum=0;
            for(int k=i+1;k<nums.size();k++)
            {
                rightSum+=nums[k];
            }

            if(leftSum==rightSum)
            {
                return i;
            }
        }
         return -1;
    }
};